main.js

```diff
@@ -143,11 +154,17 @@ function createLabel(htmlForName, labelText){
     return label;
 } 
 
-function createInput(name, inputId = name, placeholder = name, type = name){
+function createInput(value = "", name, inputId = name, placeholder = name, type = name){
     const input = createBaseElement({tagName: "input", tagId: inputId});
+    input.value = value;
     input.type = type;
     input.name = name;
     input.placeholder = placeholder;
     return input;
 }
 
+function createParagraph(paragraphId, errorText){
+    const paragraph = createBaseElement({tagName: "p", className: "error", tagId: paragraphId});
```

Comment:
Why does a Paragraph has `className: "error"` by default?

rename `createParagraph`-> `createError` or move error class outside of `createParagraph`

Reply:
Done